normandy/control/static/control/js/components/action_forms/HeartbeatForm.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { reduxForm } from 'redux-form'
+import { _ } from 'underscore'
+import { formatLabel } from '../../utils/formHelpers.js'
+
+
+class SurveyForm extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { selectedSurvey, fields, setSelectedSurvey } = this.props;
+    const surveyObject = selectedSurvey || fields['defaults'];
+    let headerText = "Default Survey Values";
+    let containerClass = 'fluid-8';
+    if (selectedSurvey) {
+      headerText = selectedSurvey.title.initialValue || "New survey"
+      containerClass += ' active';
+    }
+
+    return (
+      <div id='survey-form' className={containerClass}>
+        { selectedSurvey &&
+          <a className="return-to-defaults" href="#" onClick={(e) => {
+            e.preventDefault();
+            setSelectedSurvey()
```

Comment:
So the reason I like depending on `selectedSurvey` for what to show there is that it will toggle between showing a survey/showing the defaults automatically since it's updating with each state change.

If I add a `showDefaults` method, I'd honestly just want it to unset the `selectedSurvey` like I'm already doing. That on it's own might still be better, since `showDefaults` is more explicit/reads more semantically. Do you imagine that method doing anything different or is that the solution you had in mind?